src/axom/quest/SamplingShaper.hpp

```diff
@@ -116,19 +116,24 @@ class InOutSampler
     m_octree->generateIndex();
   }
 
-  template <int FromDim, int ToDim>
-  std::enable_if_t<ToDim != DIM, void> sampleInOutField(
-    mfem::DataCollection*,
-    shaping::QFunctionCollection&,
-    int,
-    PointProjector<FromDim, ToDim>)
-  {
-    static_assert(
-      ToDim != DIM,
-      "Do not call this function -- it only exists to appease the compiler!"
-      "Projector's return dimension (ToDim), must match class dimension (DIM)");
-  }
-
+  /**
+   * \brief Samples the in out field over the indexed geometry, possibly using a
```

Comment:
```suggestion
   * \brief Samples the inout field over the indexed geometry, possibly using a
```

consistency with other use of term